DatadogSessionReplay/SRSnapshotTests/SRSnapshotTests/Utils/SnapshotTestCase.swift

```diff
@@ -71,6 +71,11 @@ internal class SnapshotTestCase: XCTestCase {
         waitForExpectations(timeout: seconds * 2)
     }
 
+    func forEachPrivacyMode(do work: (SessionReplayPrivacy) throws -> Void) rethrows {
+        let modes: [SessionReplayPrivacy] = [.maskAll, .allowAll, .maskUserInput]
```

Comment:
Could be `SessionReplayPrivacy.allCases`, but with explicit list we can decide on which snapshots to re-generate / assert. I found it handy 🙂.